data/rulesets.js

```diff
@@ -256,7 +262,8 @@ exports.BattleFormats = {
 		},
 	},
 	hoennpokedex: {
-		effectType: 'Rule',
+		effectType: 'Banlist',
```

Comment:
What's the difference between `Rule` and `Banlist`? Why are they different `effectType`s?